### DISCUSSION TITLE
Where is the download for 86box 3.7.1?

### DISCUSSION THREAD

**Discussion by MidnightThunder** [86Box/86Box#2575] [Q&A] [answered]
The newest available download is 3.7 but the website suggests 3.7.1 has been released and comes with some critical (for me) bug fixes. Where can I download 3.7.1?

**Comment by richardg867** [comment, marked as answer]
I accidentally pushed the blog post out too early with an incomplete changelog and a broken link. Nevertheless, [3.7.1 is now out](https://github.com/86Box/86Box/releases/tag/v3.7.1).
